### PR TITLE
PMM-4993 Re-adding pmm-admin man page

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
@rnovikovP This is the new 'man-page' style reference for pmm-admin based on the output the tool itself generates. It should be seen as the starting point for expansion: we'll add more examples and hopefully use it as the 'golden source' of info to feedback into the tool's own help.
@JiriCtvrtka Please check the EXAMPLES section where the only one at the moment is 'pmm-admin status'. The output was taken from PMM-5926.
Thx

### CODE REVIEW

**Comment by JiriCtvrtka** [comment]
@PaulJacobs-percona It will be OK, after https://jira.percona.com/browse/PMM-5926 will be done.

File: source/conf.py [hunk 1]
```diff
@@ -290,7 +290,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'perconamonitoringandmanagement', u'Percona Monitoring and Management Documentation',
+    ('reference/pmm-admin', 'pmm-admin', u'Percona Monitoring and Management Documentation',
```
Comment (PaulJacobs-percona) [inline source/conf.py:293]: This creates a man page that can be added to the PMM Client distribution package somehow.

File: source/reference/pmm-admin.rst [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+   Node ID : /node_id/29b2cc24-3b90-4892-8d7e-4b44258d9309
+   PMM Server:
+    URL : https://x.x.x.x:443/
+    Version: 2.5.0
+   PMM Client:
+    Connected : true
+    Time drift: 2.152715ms
+    Latency : 465.658µs
+    pmm-admin version: 2.5.0
+    pmm-agent version: 2.5.0
```
Comment (rnovikovP) [inline source/reference/pmm-admin.rst:312]: If i remember correctly this is not yet implemented?

File: source/reference/pmm-admin.rst [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+``pmm-admin remove <service-type> [<service-name>]``
+
+``pmm-admin register``
+
+``pmm-admin status --server-url=<SERVER-URL>``
+
+``pmm-admin list --server-url=<SERVER-URL>``
+
+``pmm-admin register --server-url=<SERVER-URL>``
+   Register the current node with the PMM Server located at ``SERVER-URL``.
```
Comment (rnovikovP) [inline source/reference/pmm-admin.rst:30]: `pmm-amin config`  - one of the most important lost in this section?

File: source/reference/pmm-admin.rst (hunk 2 above)
Comment (rnovikovP) [reply source/reference/pmm-admin.rst:312]: oh .. i see. So this PR is mix: new  page + issues from Sprint

**Comment by rnovikovP** [comment]
IMO, we  should remove the  changes for  PMM-5926  and  merge this to `master` =  Ship on the website new  page 

The PMM-5926  changes - should go just separate PR  to "release branch"  but not merged until issue get to "In Doc" status to know what version it will be 2.6.1 or 2.7.0  or  2.8.0

File: source/reference/pmm-admin.rst (hunk 2 above)
Comment (PaulJacobs-percona) [reply source/reference/pmm-admin.rst:312]: Yes, two birds, one stone.

File: source/reference/pmm-admin.rst (hunk 3 above)
Comment (PaulJacobs-percona) [reply source/reference/pmm-admin.rst:30]: Yes, unintentionally.

**Pull request merged**